lib/wallaroo/core/initialization/local_topology.pony

```diff
@@ -1022,6 +1041,7 @@ actor LocalTopologyInitializer is LayoutInitializer
 // INITIALIZATION PHASES
```

Comment:
It seems like source listeners are never being told to start listening or to start the sources.

Reply:
That's the problem that I'd had after an experimental merge of the `aloc-sink3d` branch: the `ConnectorSource` wasn't being told to start listening.  The final sequence of starting listeners seems not-yet-settled, so I gave up on finishing the merge.  Over in router_registry.pony, there is a related comment: ```
      // TODO [source-migration]: Ask John if we still want to start sources here
      // given that source listeners are now initializable
```